### PR TITLE
Bug - 5219 - Fix storybook + `react-hook-form`

### LINKED ISSUE #5219: 🐛 Storybook/react-hook-form bumps causing errors
## 🐛 Bug

During yesterdays dependabot `react-hook-form` and `@storybook/*` were bumped because CI was passing but after merging all, they broke chromatic builds.

## 🦋 Expected Behaviour

Chromatic builds should be successful. 

## 📋 Steps to Reproduce

1. Run `npm i`
2. Run `npm run storybook`
3. Storybook does not build

## 🙋‍♀️ Proposed Solution

Lock in versions for now until patches are released.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] No errors in `BasicForm` component
- [ ] Storybook builds locally
- [ ] Chromatic is successful

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #5219 

## 👋 Introduction

- Locks in the version of `@storybook/*` dependencies
- Changes `react-hook-form` to only install patch releases

## 🕵️ Details

Both `@storybook/*` and `react-hook-form` bumps introduced breaking changes (despite claiming to be patch + minor releases). This locks them in to get builds working again.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Confirm chromatic built
2. Storybook runs successfully locally `npm run storybook`

### CODE REVIEW

File: frontend/admin/package.json [hunk 1]
```diff
@@ -52,13 +58,13 @@
     "@graphql-codegen/typescript-operations": "^2.5.10",
     "@graphql-codegen/typescript-urql": "^3.7.3",
     "@hydrogen-css/hydrogen": "2.0.0-beta.33",
-    "@storybook/addon-actions": "^6.5.13",
-    "@storybook/addon-essentials": "^6.5.15",
-    "@storybook/addon-links": "^6.5.15",
+    "@storybook/addon-actions": "6.5.14",
+    "@storybook/addon-essentials": "6.5.14",
+    "@storybook/addon-links": "6.5.14",
     "@storybook/addon-viewport": "^6.5.14",
```
Comment (yonikid15) [inline frontend/admin/package.json:64]: Does this need to version need to be static as well? Removing the '^'? Line 64
Just to stay in line with the other versions.

**Review by yonikid15 — COMMENTED** [review]
Everythings working on my end. Just one little question/comment about the `"@storybook/addon-viewport"` dependency.

File: frontend/admin/package.json (hunk 1 above)
Comment (esizer) [reply frontend/admin/package.json]: oops, updated in [fix viewport version number](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5220/commits/9e987424a08ab608a6ce269700be97dbdaedbaf3)

**Review by yonikid15 — APPROVED** [review]
Good to go! Ship it 📫

**Pull request merged**